src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2274,9 +2274,19 @@ static void updateAttrInNotifyCer
       //
       // FIXME P6: https://github.com/telefonicaid/fiware-orion/issues/2587
       // If an attribute has no value, then its value is not updated (neither is previousValue).
-      // However this may be problematic ... see the issue
```

Comment:
CNR line regarding this PR missing. Suggestion:

```
Fix: null not working in q/mq filter in subscriptions (#2998)
```

Reply:
Fixed in ba413e8